### DISCUSSION TITLE
As we add mibs ourselves from products, should we update the generator documentation with the information?

### DISCUSSION THREAD

**Discussion by EStork09** [prometheus/snmp_exporter#1041] [General]
I know there are a few MIBs already bundled with the generator, should we make PRs to get new sets added, or should we be doing that independently? (Such as adding DELL, UCS, HP, etc)

**Comment by SuperQ** [comment]
I've been wanting to build a MIB/generator knowledge and example config base separately from the exporter.

https://github.com/prometheus-community/snmp

Now that the exporter module and auth is split, as well as supporting multiple input snmp.yml config files, we can do a module-per-generator file and build up some automation around generating output files that can be more copy-n-paste for everyone to use.

**Reply by EStork09** [reply]
That sounds great, I can wait for that to get more ground and hopefully remember information and create PRs for anything I figure out in the future. Thanks for the quick response!

**Reply by SuperQ** [reply]
Sadly, I don't have a ton of time to work on this. This is why the repo is still empty after years.
